README.md

```diff
@@ -1,2 +1,195 @@
+
+> It will run local webserver or any webhosting cpanel 
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://mozilla.com/">Any web broweser/mostly mozila/firefox</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Php and Html server like apache/wamp</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **The project display hellow microverse**
+- **Bult with pure HTML**
+- **Bult using CSS**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Download the application to your local computer.
+> cd into it.
+>npm update
+>stlelint "**/*.{css/scss
+>Add custom code 
+
+
+### Prerequisites
+>Node Js
+>code editor 
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+  cd my-folder
+  git clone https://github.com/samuelkinyanjui/Hello-Microverse
+
+
+### Install
+
+Install this project with:
+copy the folder to Wamp web folder
+
+### Usage
+
+Launch your server
+
+
+
+### Run tests
+
+open the home/index file
+
+
+### Deployment
+
+You can deploy this project using:
+
+Use amazon or inhouse hosting for deployment
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a
+> Microverse,Samuel
+
+👤
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/samuelkinyanjui/Hello-Microverse)
+- Twitter: [@twitterhandle](https://twitter.com/sax80kinya)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/samuelkinta)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+>Android based site
+I will addedmore android and IOS supported tempalte
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+Write an email to sax80kinya@gmail.com
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Use the project to buld your own website
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+
+I would like to thank...all microverse student
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+
+
+What is the main tool used to make this application?
+
+  HTML
+
+-How can we use the template?
+you just clone it
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+https://github.com/samuelkinyanjui/Helloworldrpt/blob/main/LICENSE
```

Comment:
Dear Johnadibe,

I am not able to understand your comment.
I already added the license file to the main branch

Reply:
Hello @samuelkinyanjui, Could we jump on a brief call so that I can explain better?😊

Reply:
ok. am online and ready for the call

Reply:
Or how do I reach you for the call

Reply:
Hello @samuelkinyanjui, I'm sorry for the delayed response.😔 I have sent a message to you on slack. Please kindly check and confirm so that we can jump on the brief call.😊